lib/BridgeList.php

```diff
@@ -178,10 +179,14 @@ private static function getFooter($totalBridges, $totalActiveBridges, $showInact
 
 		}
 
+		if($donationsAllowed) {
+			$donation = ' ~ <a href="http://www.google.com">Donate</a>';
```

Comment:
Wnat is expected here?

Reply:
Ah, thats right. Thats the rss-bridge-general-donation-url, so not bridge specific but for rss-bridge itself. You need to define that link